### PR TITLE
use new version of invoker plugin with support of node path naming

### PR DESCRIPTION
**Pull request by olamy**
Signed-off-by: olivier lamy <olamy@apache.org>

### CODE REVIEW

File: jenkins-plugin/pom.xml [hunk 1]
```diff
@@ -87,6 +87,66 @@
         </exclusion>
       </exclusions>
     </dependency>
+    <!-- we need to force some dependencies introduced by plugin depending on maven-plugin such jacoco or invoker-->
```
Comment (cyrille-leclerc) [inline jenkins-plugin/pom.xml]: Why are those `<dependencies>` and not `<dependencyManagement>`?

Comment (olamy) [reply jenkins-plugin/pom.xml]: This will not work with enforcer-plugin/RequireUpperBoundDeps and those dependencies version need to be enforced.

Comment (cyrille-leclerc) [reply jenkins-plugin/pom.xml]: So this means that the mechanism to build jenkins plugins requires to list dependencies that are in fact not used directly by the plugin itself? 

This means that we have to declare "commons-digester" as a dependency even if the code of the pipeline-maven-plugin does not directly reference any class of "commons-digester"?

Comment (olamy) [reply jenkins-plugin/pom.xml]: mostly it is (because of the enforcer rule).
But I will change those dependencies as optional to make enforcer happy.....

**Pull request merged**